File: working/temas/30.4.1.rb
class Frigo
    def initialize(color, temperatura)
        @color = color
        @temperatura = temperatura
    end

    def to_s
        return "Color #{@color} | #{@temperatura}ºC"
    end

    def fija_temperatura(nueva_temperatura)
        if nueva_temperatura < 0
            raise ArgumentError.new("no puede ser < 0")
        end
        @temperatura = nueva_temperatura
    end
end


f1 = Frigo.new("rojo", 18)
puts f1

f1.fija_temperatura(-1)

File: working/temas/30.4.1.py
class Frigo:
    def __init__(self,color, temperatura):
        self.__color = color
        self.__temperatura = temperatura

    def __str__(self):
        return f"Color {self.__color} | {self.__temperatura}ºC"

    def fija_temperatura(self,nueva_temperatura):
        if nueva_temperatura < 0:
            raise ValueError("no puede ser < 0")
        self.__temperatura = nueva_temperatura


f1 = Frigo("rojo", 18)
print(f1)

f1.fija_temperatura(-1)
